src/client/components/Navigation/Topnav.less

```diff
@@ -86,6 +86,18 @@
     }
   }
 
+  &__popover {
+    & li {
+      display: block;
+      
+      @media screen and (min-width: 991px) {
```

Comment:
You should try to use the constants from `busy/src/client/styles/modules/variables.less`

Reply:
I have thought of that. Actually, that's how I did it the first time, locally.

But those do not match the `antd` responsive break-points, and it does not behave good, as we might expect.

For instance, the `replies` link will show up on the top (normal version) dropdown between 991px and 998px, which was not intended.

Reply:
That same declaration is already found in that file on another occasion. I agree that it should not be hard coded like that.

Do you have any suggestions about this? Maybe make another media var for `antd`? Or, maybe should we change the `@medium` var to match the `antd` break-point?

Reply:
I think it would be best to make another breakpoint then for antd, maybe `@antd-medium`